packages/components/pages/f-account-info/package.json

```diff
@@ -40,20 +40,22 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-globalisation": "1.0.0"
+    "@justeat/f-card-with-content": "1.0.0",
+    "@justeat/f-globalisation": "1.0.0",
+    "@justeat/f-vue-icons": "3.3.0"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.2.0",
     "vuex": ">=3.5.1"
   },
   "devDependencies": {
     "@justeat/f-wdio-utils": "0.5.0",
-    "node-sass-magic-importer": "5.3.2",
     "@samhammer/vue-cli-plugin-stylelint": "2.1.0",
     "@vue/cli-plugin-babel": "4.5.15",
     "@vue/cli-plugin-eslint": "3.9.2",
     "@vue/cli-plugin-unit-jest": "4.5.15",
     "@vue/test-utils": "1.2.2",
-    "axios": "0.24.0"
+    "axios": "0.24.0",
```

Comment:
Are we using axios, or is it part of a peer dependency perhaps?

Reply:
Looks like this was added during the initial `0.1.0` scaffold - https://github.com/justeat/fozzie-components/commit/de79a7fe4c0c6c1f592d4191c1dfbfaf3c7ba4ad#diff-19f4ec164c06c2a8249537ffc493d968cb3acde26d8d42d96ec58054fbc9dec4R57 Will remove it,